examples/baby_name_generator/requirements.txt

```diff
@@ -1,2 +1,3 @@
 agenta
-openai
\ No newline at end of file
+openai
+pydantic>=2
```

Comment:
do we need this to work?

Reply:
No, it was part of the QA that led me to discover [this](https://github.com/Agenta-AI/agenta/pull/1705#discussion_r1624231626).

Reply:
should this now be reverted